babel.config.js

```diff
@@ -0,0 +1,23 @@
+const { version } = require('./package.json')
+module.exports = {
+  presets: ['cozy-app'],
+  env: {
+    transpilation: {
+      plugins: [
+        [
+          'css-modules-transform',
+          {
+            extensions: ['.styl'],
+            preprocessCss: './preprocess',
+            extractCss: './transpiled/react/stylesheet.css',
```

Comment:
Pourquoi le mettre dans `transpiled/react/stylesheet.css`, ca pourrait ne pas être un breaking change non ?

Reply:
Putting the stylesheet file at the root of `transpiled/` forces all applications to import `cozy-ui/transpiled/stylesheet.css`, so they can't use the alias `'cozy-ui/react': 'cozy-ui/transpiled/react'` and when we will put back transpiled components into `react/` we will push a breaking change for all apps using it.
To avoid a breaking change right now we could copy the file into `transpiled/stylesheet.css` but since it's used only in Banks and Drive I think we can handle it using a new major version imho